README.md

```diff
@@ -1 +1,66 @@
-# html-app-starter
\ No newline at end of file
+# HTML App Starter
+
+A template for starting a web app, particularly for the HTML5 zip content type for Sushi Chefs! Attempts to format text according to the [Kolibri Style Guide](http://kolibribeta.learningequality.org/style_guide#/).
+
+File structure:
+
+```
+.
+├── index.html
+├── css
+│   └── styles.css
+├── js
+│   └── scripts.js
+├── img
+│   └── kolibri.png
+└── fonts
```

Comment:
I wouldn't include the fonts in the base template. It would add 800k (476K zipped) to each HTML5App.